src/blocks/product-card.css

```diff
@@ -0,0 +1,102 @@
+.card {
+  font-family: "Roboto", sans-serif;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  font-weight: 500;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+  padding: 32px 16px 16px;
+  align-items: center;
+
+}
+
+.card__img-box {
+  display: flex;
+  margin-bottom: 40px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  width: 100%;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: #060b35;
+  margin-bottom: 4px;
+}
+
+.card__product-code {
+  align-self: flex-start;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.card__review {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.card__review-text {
+
+  font-style: normal;
+  font-size: 10px;
+  font-weight: 400;
+}
+
+.card__price-box {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+  width: 100%;
+}
+
+.card__price-text {
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__price-price {
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  text-decoration: none;
+  background: #00acdc;
+  border-radius: 5px;
+  border: 1px solid #00acdc;
+  color: #fff;
+  height: 40px;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-transform: uppercase;
+}
+
+.card__button:hover {
+  background: #fff;
+  border: 1px solid #00acdc;
```

Comment:
Set transparent border to your link and change only its color in :hover.

See more details in checklist, point 9.